build.xml

```diff
@@ -6,6 +6,16 @@
   <description>
       Build file for Redistricting project investigation
   </description>
+  
+    <!--This sets up the build.properties file. Displays ant verison, site name, release verison, and name of the release-->
```

Comment:
This comment doesn't seem appropriate. The build.properties file needs to be set up outside the automated build process. This target appears to only display the contents of an existing build.properties file.